### PR TITLE
[drogon] Update to 1.8.2

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

File: ports/drogon/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "drogon",
-  "version-semver": "1.8.1",
+  "version-semver": "1.8.2",
```
Comment (MonicaLiu0311) [inline ports/drogon/vcpkg.json:3]:
```suggestion
  "version": "1.8.2",
```

Comment (MonicaLiu0311) [reply ports/drogon/vcpkg.json:3]: It should be `version`, because there is no `semver` related information upstream.

**Comment by MonicaLiu0311** [comment]
All features are tested successfully in the following triplet:
x86-windows
x64-windows
x64-windows-static

**Comment by MonicaLiu0311** [comment]
There is a conflict in the file, after making any modification, please run `./vcpkg x-add-version drogon` to generate the corresponding json; If you are modifying the same version, please use `./vcpkg x-add-version drogon --overwrite-version` to update git-tree.

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**